### ISSUE TITLE
Provide typical usage of --tools

### ISSUE THREAD

**Issue by ameynert** [nf-core/sarek#848]
### Description of feature

The regex for describing how to specify ``--tools`` is not easy to understand for non-command line users. Adding some tool combination examples for typical usage would be helpful. Also suggest specifying the exact capitalization for each tool with the tool listing, not just in the regex.

**Comment by FriederikeHanssen** [comment]
so something like 

```
example usage:

--tools strelka,snpeff

```

?

**Comment by ameynert** [comment]
Yes, just like that. Some starting point recommendations for e.g. germline vs tumour/normal, WGS or exome would also be useful.

**Comment by ameynert** [comment]
How about linking to that from the usage?

**Comment by FriederikeHanssen** [comment]
:D hah yes just updated my comment. sounds like a good idea 👍

**Comment by asp8200** [comment]
Hmmm 🤔 So on the abovementioned table strelka is called "Strelka2", but one is suppose to write `--tools strelka` on the command line?

Similarly, the command for "GATK Haplotypecaller` is `--tools haplotypecaller`.

Moreover, it is not clear to me whether one needs to use the capitalization of the names of the tools on the command line as it is done on in the table, that is, the table lists, for instance, `FreeBayes` but should one write `--tools FreeBayes` on the command line?  (I happen to know that `--tools freebayes` will work, but will `--tools FreeBayes` also work?)

**Comment by maxulysse** [comment]
we should really fix #892, that will help with the spelling of the tools
